styles.css

```diff
@@ -0,0 +1,98 @@
+nav {
+  background-color: #2B78E4;
+  border-bottom: 2px solid black
+}
+
+.pagetitle {
+  color: #FFFFFF;
+  font-family: "Comic Sans MS";

```

Comment:
Comic Sans everywhere! I think you are interpreting the mockup too literally. Bootstrap should be all sleek and cool and Helvetica!